gotv_transportation.md

```diff
@@ -0,0 +1,169 @@
+          * [Language](#language)
+          * [Accessibility](#accessibility)
+    * [Vehicle Fields](#vehicle-fields)
+      * [VehicleSeat](#vehicle-seat)
+    * [Assignment Fields](#assignment-fields)
+    * [Interval Fields](#interval-fields)
+    * [Location Fields](#location-fields)
+* [Scenarios](#scenarios)
+    * [Scenario A centrally implemented, web-based GOTV system](#scenario-a-centrally-implemented-web-based-gotv-system)
+    * [Scenario: A federated implementation](#scenario-a-federated-implementation)
+
+GOTV Transportation represents the collection of types used to describe and satisfy specific demands for transportation to a voting or registration location. The model consists of Riders, Demands, Vehicles, Drivers, and Assignments, as well as other supporting types. At the highest level, the goal of a GOTV Transportation system is to match drivers and vehicles with riders' demands, making as many assignments as possible to drive turnout.
+
+Demands represent a demand for transportation to a voting location by one or more Riders. Each rider may be a single voter, and demands might be associated with more than one rider, representing a group of voters or, in some cases, a group of voters as well as non-voters. As an example of this last category, consider the combination of a voter and his fulltime healthcare worker, or a voting parent and a minor child. Demands indicate the set of locations which could satisfy them, as well as the times during which they are valid and other constraints on how it could be solved. A constraint might be, for instance, wheelchair accessibility, or the rider's preferred language.
+
+Vehicles are used to transport riders, and have an associated driver. Vehicles and drivers both have various state capabilities and capacities. For example, an SUV may have capacity for 4 riders and be wheelchair accessible. It's driver may be a Spanish speaker and male. In some cases, a vehicle may be operated by more than one person, such as when two people volunteer to share the load of driving, or to feel more comfortable in picking up and traveling with strangers. In these cases, we do not attempt to model the pair of drivers, but do expect the driver to adjust the seating capacity of the vehicle in order to account for the extra driver.
+
+For more details on how a system might be built on the data types proposed here, see the [Scenarios](#scenarios).
+
+## Fields
+
+### Demand Fields
+
+| Name  | Type  | Description
+|-----  |-----  |------
+| availability  | Interval[]  | The Intervals during which this Demand can be satisfied
+| locations | Location[] | The locations to which the associated Riders may be transported
+| riders  | Rider[] | The Riders generating this Demand
+| coordinating_rider | Rider* | If set, then communications about an assigned Ride should be directed towards this Rider. It is an error if this value is set but is not associated with an element of `riders`
+
```

Comment:
```suggestion
| ride_coordinator | Supporter | This would be someone like a staff member at a senior center who would be coordinating rides for residents, helping riders load into the vehicles, etc, but not traveling with the rider. They would get notifications for the riders they are associated with. 
```

Reply:
Thanks -- I added it here as `external_coordinator` and just changed your wording a little bit. Reasonable start?